### DISCUSSION TITLE
`npx tree-sitter test` behaves differently across OSes

### DISCUSSION THREAD

**Discussion by mrdgo** [tree-sitter/tree-sitter#3669] [Q&A] [answered]
If you take a look at [this PR](https://github.com/nushell/tree-sitter-nu/pull/113), I added an external scanner to our tree-sitter parser.

Weirdly enough, the github actions only fail on windows. And `npx tree-sitter test --debug` provides [some more output](https://github.com/nushell/tree-sitter-nu/actions/runs/10989810751/job/30508749508?pr=113), but I can't parse the underlying error from it. Can someone here?

**Comment by amaanq** [comment]
Line endings are different on Windows and Linux.

**Reply by mrdgo** [reply, marked as answer]
Thanks for your answer.

Apparently, some dependencies were out of date. No problem on unix-systems but on windows. Either way, thanks for the quick hint!
